content/pages/office-policy.md

```diff
@@ -0,0 +1,42 @@
+Title: Office Policy
+Category: pages
+Status: hidden
+Slug: office-policy
+
+(Version 2.0 March 8th 2015)
+
+# Personnel #
+
+* If the office is open for office hours, drop-ins or for any other reason, a MINIMUM of ONE committee member of WiCS must be present in the office at all times.
+* All WiCS members occupying the office are responsible for complying with and enforcing the Code of Conduct.
+
+# Cleanup: #
+
+* The last WiCS staff member to occupy the office is responsible for cleanup.
```

Comment:
This was copied straight from the drive. The rest of the bullets seem to be specific tasks, maybe that's why they were grouped in to a sub list. I didn't want to change the content by myself. Should I align them instead?